src/main/java/htsjdk/tribble/index/Index.java

```diff
@@ -73,41 +73,42 @@
     /**
      * Writes the index into a file.
      *
+     * Default implementation delegates in {@link #write(Path)}
```

Comment:
Thanks for this, I really appreciate to have comments to improve my English skills. Done!